<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ManageProfile</class>
 <widget class="QDialog" name="ManageProfile">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>544</width>
    <height>524</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QGroupBox" name="groupBox_updateProfile">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>50</y>
     <width>291</width>
     <height>351</height>
    </rect>
   </property>
   <property name="title">
    <string>Update Profile</string>
   </property>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>40</y>
      <width>81</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_updateProfile">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>290</y>
      <width>131</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Update Profile</string>
    </property>
   </widget>
   <widget class="QWidget" name="layoutWidget_10">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>40</y>
      <width>241</width>
      <height>231</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_4">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QLabel" name="label_weight">
         <property name="text">
          <string>Weight (kg):</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="lineEdit_newWeight"/>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_4">
       <item>
        <widget class="QLabel" name="label_sex">
         <property name="text">
          <string>Sex (M/F):</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="lineEdit_newSex"/>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QLabel" name="label_age">
         <property name="text">
          <string>Age:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="lineEdit_newAge"/>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <item>
        <widget class="QLabel" name="label_height">
         <property name="text">
          <string>Height (cm):</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="lineEdit_newHeight"/>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QLabel" name="label_currName">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>140</y>
     <width>181</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Name:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>40</y>
     <width>171</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Press to view Current Information</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_currWeight">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>220</y>
     <width>181</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Weight (kg):</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_currSex">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>310</y>
     <width>181</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Sex (M/F): </string>
   </property>
  </widget>
  <widget class="QLabel" name="label_currAge">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>180</y>
     <width>181</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Age: </string>
   </property>
  </widget>
  <widget class="QLabel" name="label_currHeight">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>260</y>
     <width>181</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Height (cm):</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>80</y>
     <width>111</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>View Info</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_stepGoal">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>360</y>
     <width>181</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Step Goal:</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_updateStepGoal">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>460</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Update</string>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>460</y>
     <width>281</width>
     <height>31</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_5">
    <item>
     <widget class="QLabel" name="label_4">
      <property name="text">
       <string>New Step Goal:</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="lineEdit_newStepGoal"/>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
